docs: fix grammar mistakes and typos across the documentation (#637)
fluent/fluentd-docs-gitbook: how-to-guides/cep-norikra.md

The `<match>` section specifies the glob pattern used to look for the matching tags. If the tag of a log is matched, the respective `match` configuration is used \(i.e. the log is routed accordingly\).

> The `norikra` attribute specifies the Norikra server's RPC host and port \(default: `26571`\). By `target_map_tag true` and `remove_tag_prefix data`, `out_norikra` handles the rest of tags \(e.g. `foo` for `data.foo`\) as the target, which is the name of the set of events the same as table name of RDBMS.

Reviewer: It simplifies the longer sentence.

```suggestion
The `norikra` attribute specifies the Norikra server's RPC host and port \(default port: `26571`\). With `target_map_tag true` and `remove_tag_prefix data`, `out_norikra` handles the rest of tags \(e.g. `foo` for `data.foo`\) as the target. A target is the name of a set of events, similar to a table name in an RDBMS.
```
